lib/dh.js

```diff
@@ -100,16 +89,23 @@ function DH(prime, generator, malleable) {
   this._primeLen = prime.length;
   this._pub = void 0;
   this._priv = void 0;
-  
+  this._primeCode = void 0;
```

Comment:
We can also just change then to undefined

On Mon, Oct 26, 2015, 7:25 PM Daniel Cousens <notifications@github.com>
wrote:

> In lib/dh.js
> <https://github.com/crypto-browserify/diffie-hellman/pull/18#discussion_r43067638>
> :
>
> > @@ -100,16 +89,23 @@ function DH(prime, generator, malleable) {
> >    this._primeLen = prime.length;
> >    this._pub = void 0;
> >    this._priv = void 0;
> > -
> > +  this._primeCode = void 0;
>
> This is really uncommon, could you put a comment to explain what the void
> 0 achieves here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crypto-browserify/diffie-hellman/pull/18/files#r43067638>
> .
>